user_profile.py

```diff
@@ -336,18 +387,17 @@ def post(self):
                             # create a list of new user instance
                             new_user = UserInformation(first_name=firstName, middle_name=midName, last_name=lastName, age=age, date_of_birth=birthDay, address_line_1=firstAddress, address_line_2=secondAddress, city=city, province=province, country=country, postal_code=postalCode, gender=gender, religion=religion, profile_image=profile_image, user_bio=user_bio, interests=user_interest, education_institutions=education_institution, education_majors=education_majors, education_degrees=education_degrees, graduation_date=graduation_date, identification_option=identification_option, identification_material=identification_material, user_id=user_id)
 
-
                             # add new user into users model
                             db.session.add(new_user)
                             # commit the change to the database -> 201 if successful
                             db.session.commit()
 
-                            response_data = {"user_info": json.dumps(new_user)}
+                            response_data = {"message" : f"User's information has been added successfully!"}
```

Comment:
This is a successful POST request, we do not need message here, a POST request with status 201 indicates the resource (in this case user information) is created successfully.

Let's serializes the user information data (but do not include the `id` of the user information) , and include it in the response. For context: frontend will use the data created to display user information